src/Sylius/Bundle/AttributeBundle/Form/EventSubscriber/BuildAttributeValueFormSubscriber.php

```diff
@@ -75,17 +78,52 @@ public function preSubmit(FormEvent $event)
         $form = $event->getForm();
         $attribute = $this->attributeRepository->find($attributeValue['attribute']);
 
-        $this->addValueField($form, $attribute);
+        $this->addValueField($form, $attribute, $attributeValue);
```

Comment:
Do not pass $attributeValue in this case, because looking above on line 79 you know that it's an array and not an object. 

You only have the object in `preSetData()` so this is where it should be passed in